cumulusci/core/github.py

```diff
@@ -117,8 +117,8 @@ def get_github_api_for_repo(keychain, repo_url, session=None):
     elif GITHUB_TOKEN:
         gh.login(token=GITHUB_TOKEN)
```

Comment:
Leaving this untouched because of backwards compatibility. This will still also support github enterprise connections, but only for single connections. The modifications we are adding overall to support GitHub Enterprise Server, will allow greater flexibility without changing these lines..